From 1e849719a1035f34f44873a44a3d6266a59a5f1d Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@arm.com>
Date: Thu, 18 Jan 2024 12:59:37 +0000
Subject: [PATCH] configure: don't disable documentation if latex2man isn't
 present

latex2man is only needed to generate the manpages from the LaTeX
sources, but as the git repository has pregenerated manpages in (and so
do the tarballs) this tool is only needed if the sources have been
changed.

Instead of turning off documentation,  just warn that the manpages
cannot be rebuilt.

Signed-off-by: Ross Burton <ross.burton@arm.com>
---
 configure.ac | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 406f3773c..7744251b5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -297,8 +297,7 @@ AC_ARG_ENABLE([documentation],
 AC_MSG_RESULT([$enable_documentation])
 AC_PATH_PROG([LATEX2MAN],[latex2man])
 AS_IF([test "x$LATEX2MAN" = "x" && test "x$enable_documentation" != "xno"], [
-  AC_MSG_WARN([latex2man not found. Install latex2man. Disabling docs.])
-  enable_documentation="no";
+  AC_MSG_WARN([latex2man not found, unable to regenerate the manpages])
 ])
 AM_CONDITIONAL([CONFIG_DOCS], [test x$enable_documentation != xno])
 AM_COND_IF([CONFIG_DOCS], [AC_CONFIG_FILES([doc/Makefile doc/common.tex])])
